frontend/app/src/router.tsx

```diff
@@ -335,6 +335,330 @@ const routes: RouteObject[] = [
           },
         ],
       },
+
+      // V1 Routes - These are explicitly listed for IDE support
+
+      {
+        path: '/v1/auth',
```

Comment:
I don't think `/v1/auth` endpoints make sense, because we won't have the tenant version information at the auth layer.

Reply:
ah yeah that's reasonable - I was thinking we'd want to version the whole app, but this makes sense. I'll unwind these ones!